de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/PowerSummaryTest.java

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.swtbot.test;
+
+
+import static org.junit.Assert.assertEquals;
+
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotTable;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotTreeItem;
+import org.junit.Before;
+import org.junit.Test;
+
+import de.dlr.sc.virsat.concept.unittest.util.ConceptXmiLoader;
+import de.dlr.sc.virsat.model.dvlm.concepts.Concept;
+import de.dlr.sc.virsat.model.extension.budget.power.model.PowerEquipment;
+import de.dlr.sc.virsat.model.extension.budget.power.model.PowerState;
+import de.dlr.sc.virsat.model.extension.budget.power.model.PowerSummary;
+import de.dlr.sc.virsat.model.extension.ps.model.ConfigurationTree;
+import de.dlr.sc.virsat.model.extension.ps.model.ElementConfiguration;
+
+public class PowerSummaryTest extends ASwtBotTestCase {
+	private SWTBotTreeItem repositoryNavigatorItem;
+	
+	private SWTBotTreeItem configurationTree;
+	private SWTBotTreeItem elementConfiguration1;
+	private SWTBotTreeItem elementConfiguration2;
+
+	private static final Double AOCS_POWER = 20.0;
+	private static final Double AOCS_DUTYCYCLE = 20.0;
+	
+	private static final Double DATA_HANDLING_POWER = 50.0;
+	private static final Double DATA_HANDLING_DUTYCYCLE = 50.0;
+
+	private SWTBotTreeItem powerEquipment1;
+	private SWTBotTreeItem powerEquipment2;
+	private SWTBotTreeItem powerSummary;
+
+	private Concept conceptPower;
+
+	
+	@Before
+	public void before() throws Exception {
+		super.before();
+		//create necessary items for the test
+		conceptPower = ConceptXmiLoader.loadConceptFromPlugin(de.dlr.sc.virsat.model.extension.budget.power.Activator.getPluginId() + "/concept/concept.xmi");
+		
+		repositoryNavigatorItem = bot.tree().expandNode(SWTBOT_TEST_PROJECTNAME, "Repository");
+
+		configurationTree = addElement(ConfigurationTree.class, conceptPs, repositoryNavigatorItem);
+
+		elementConfiguration1 = addElement(ElementConfiguration.class, conceptPs, configurationTree);
+		rename(elementConfiguration1, "AOCS");
+		
+		powerEquipment1 = addElement(PowerEquipment.class, conceptPower, elementConfiguration1);
+		openEditor(powerEquipment1);
+
+		bot.button("Add PowerState").click();
+		bot.table().select("powerValues");
+		bot.button("Drill-Down").click();
+		
+		renameField(PowerState.PROPERTY_POWER, AOCS_POWER + "");
+		renameField(PowerState.PROPERTY_DUTYCYCLE, AOCS_DUTYCYCLE + "");
```

Comment:
Can we remove the empty Strings?

Reply:
Well, it's the simplest way that I know how to concert the double values to strings...

Reply:
Sorry, you are right

Reply:
I actually agree that using `+ ""` to convert to String isnt the nicest way.
I would suggest `String.valueOf(AOCS_POWER)`.  It may be a little bit longer in terms of number of characters, but this makes it really clear for the reader whats being done.

Reply:
Good idea